### DISCUSSION TITLE
Vitest Documetation Translation

### DISCUSSION THREAD

**Discussion by nazarepiedady** [vitest-dev/vitest#2307] [Q&A]
### Goal
In this discussion, I invite all community members to manifest their
fells about the languages options to read and consume the docs so
if you have an interest in helping translate the documentation to your 
language please write it down to everyone know your want to help.

I also invite the maintainers to write or reference the guidelines for the
translations works, and provide some recommendations to the people.

@antfu @Aslemammad @DerYeger @elonehoo @sheremet-va 
@Demivan @edimitchel @patak-dev @zxch3n

**Comment by Naoto-Ida** [comment]
Hi! Coming here from working a little bit on the Japanese translations for Vite.
Would love to support translation work, since I love Vitest and would be great to have it reach a larger audience!

**Reply by nazarepiedady** [reply]
Nice, @Naoto-Ida you are a noble programmer. I recommend you fork the main repository and organize a team to help you to translate the documentation and maintain the community informed about the progress of the translation.

I also help translating the vite documentation, but I did it completly alone because I had a little dificulty finding people interested to help.

**Reply by Naoto-Ida** [reply]
AFAIK there's the [Chinese documentation](https://github.com/vitest-dev/docs-cn) that we could work off of. Shall we then fork these as docs-pt and docs-ja, then go from there?

**Reply by elonehoo** [reply]
Wow, that would be awesome, very much looking forward to your Chenpin

**Reply by nazarepiedady** [reply]
@Naoto-Ida I believe that would be great because helps to ease the work.
